.github/workflows/ci.yml

```
@@ -632,6 +632,61 @@ jobs:
         run: env
         if: ${{ always() }}
 
+  macos-monterey:
+    name: "x86_64: macOS Monterey, ctime_tests"
+    # See: https://github.com/actions/runner-images#available-images.
+    runs-on: macos-12
```

If we remove the Ventura jobs: 
```suggestion
  macos-native:
    name: "x86_64: macOS Monterey"
    # See: https://github.com/actions/runner-images#available-images.
    runs-on: macos-12 # Use M1 once available https://github.com/github/roadmap/issues/528
```

---

Thanks! [Done](https://github.com/bitcoin-core/secp256k1/pull/1412#issuecomment-1695937918).